exercises/largest-series-product/largest_series_product_test.go

```diff
@@ -2,27 +2,39 @@ package lsproduct
 
 import "testing"
 
-const targetTestVersion = 3
+const targetTestVersion = 4
 
-func TestLargestSeriesProduct(t *testing.T) {
+func TestTestVersion(t *testing.T) {
 	if testVersion != targetTestVersion {
 		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
 	}
+}
+
+func TestLargestSeriesProduct(t *testing.T) {
 	for _, test := range tests {
 		p, err := LargestSeriesProduct(test.digits, test.span)
-		switch {
-		case err != nil:
-			if test.ok {
+		if !test.ok { // expect error
+			// check if err is of error type
+			var _ error = err
```

Comment:
To clarify, the type check `var _ error = err` is verified at compile time, and it doesn't matter where in the file it is, as long as `err` is defined in that scope. My thought was that putting it closer to the place where `err` is originally defined would make this easier to read. As it is it doesn't change the behavior by having it here. I'll go ahead and merge this.